packages/core/upload/admin/src/pages/App/components/Filters.jsx

```diff
@@ -1,7 +1,7 @@
-import React, { useRef, useState } from 'react';
+import React from 'react';
```

Comment:
```suggestion
import * as React from 'react';
```
just to respect our standard